controllers/ovs/flowsconfig_ovnk_reconciler.go

```diff
@@ -50,9 +51,16 @@ func (c *FlowsConfigOVNKController) Reconcile(
 func (c *FlowsConfigOVNKController) updateEnv(ctx context.Context, target *flowslatest.FlowCollector, desiredEnv map[string]string) error {
 	rlog := log.FromContext(ctx, "component", "FlowsConfigOVNKController")
 
-	ds, err := c.getDaemonSet(ctx)
-	if err != nil {
-		return err
+	ds := &appsv1.DaemonSet{}
+	if err := c.client.Get(ctx, types.NamespacedName{
+		Name:      c.config.DaemonSetName,
+		Namespace: c.config.Namespace,
+	}, ds); err != nil {
+		if kerr.IsNotFound(err) && !target.Spec.UseIPFIX() {
+			// If we don't want IPFIX and ovn-k daemonset is not found, assume there no ovn-k, just succeed silently
+			return nil
```

Comment:
can we add a log info message here just stating non ovn-kubernetes mode ?

Reply:
sure